spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-web-groovy-templates/src/main/java/smoketest/groovytemplates/SampleGroovyTemplateApplication.java

```diff
@@ -29,16 +28,6 @@ public MessageRepository messageRepository() {
 		return new InMemoryMessageRepository();
 	}
 
-	@Bean
-	public Converter<String, Message> messageConverter() {
```

Comment:
I refactored that to be a lambda and suddenly it started failing. I then decided to extract the Converter into a separate class.